clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -610,13 +610,17 @@ DPCT_ENUM_OPTION(
             "Generate code with sycl:: namespace, applied only for "
             "SYCL math functions.\n"
             "Cannot be used with cl or sycl values.",
-            false)),
+            false),
+        DPCT_OPTION_ENUM_VALUE("syclcompat",
+                               int(ExplicitNamespace::EN_SYCLCompat),
+                               "Generate code with syclcompat:: namespace.",
```

Comment:
```suggestion
                               "Use SYCLcompat header-only library (syclcompat:: namespace) instead of DPCT helper functions (dpct:: namespace) to assist the migration of input source code.",
```